devfile.yaml

```diff
@@ -1,14 +1,23 @@
-apiVersion: 1.0.0
 metadata:
   name: kubernetes-api
+projects:
+  - name: kubernetes-api
+    source:
+      location: 'https://github.com/devfile/kubernetes-api.git'
+      type: git
+    clonePath: src/github.com/devfile/kubernetes-api
```

Comment:
it's go mod based, right> let's move it to projects root?

Reply:
well, not possible since the Go language server doesn't support using the go module mode in Che Theia unfortunately.